Pinta.Core/ImageFormats/TgaExporter.cs

```diff
@@ -95,20 +93,19 @@ public void Export (Document document, Gio.File file, Gtk.Window parent)
 		using var file_stream = new GioStream (file.Replace ());
 		using var writer = new BinaryWriter (file_stream);
 
-		TgaHeader header = new TgaHeader {
-			idLength = (byte) (ImageIdField.Length + 1),
-			cmapType = 0,
-			imageType = 2, // uncompressed RGB
-			cmapIndex = 0,
-			cmapLength = 0,
-			cmapEntrySize = 0,
-			xOrigin = 0,
-			yOrigin = 0,
-			imageWidth = (ushort) surf.Width,
-			imageHeight = (ushort) surf.Height,
-			pixelDepth = 32,
-			imageDesc = 8 // 32-bit, lower-left origin, which is weird but hey...
-		};
+		TgaHeader header = new (
+			idLength: (byte) (ImageIdField.Length + 1),
+			cmapType: 0,
+			imageType: 2,
```

Comment:
Can you restore the two comments (`uncompressed ...` and `32 bit...`) which were lost here?

Reply:
Sure. Sorry for not noticing. I've re-added them now